cms/envs/production.py

```diff
@@ -359,6 +359,8 @@
 AWS_QUERYSTRING_EXPIRE = 7 * 24 * 60 * 60  # 7 days
 AWS_S3_CUSTOM_DOMAIN = AUTH_TOKENS.get('AWS_S3_CUSTOM_DOMAIN', 'edxuploads.s3.amazonaws.com')
 AWS_S3_ENCRYPTION = ENV_TOKENS.get('AWS_S3_ENCRYPTION', False)
+AWS_S3_REGION_NAME = ENV_TOKENS.get('AWS_S3_REGION_NAME')
+AWS_S3_SIGNATURE_VERSION = ENV_TOKENS.get('AWS_S3_SIGNATURE_VERSION')
```

Comment:
@kaizoku, shouldn't these have default values that match the current defaults in the platform?

Reply:
The default for these is `None` in boto, so I thought it's best to set these to `None` if not specified.
Should we specify `us-east-1` for region and `v2` for signature version since those are the typical defaults for edX?

Reply:
@kaizoku, `None` is okay 👍🏼 But edX may ask us to add it explicitly in the upstream PR.